src/reference-designs/tko-on-vmc-aws.md

```diff
@@ -26,383 +41,361 @@ The following additional benefits are enabled by the Elastic Network Interface t
+- [**Helm-based deployment**](https://goharbor.io/docs/2.4.0/install-config/harbor-ha-helm/) to a Kubernetes cluster - this installation method may be preferred for customers already invested in Helm.
 
-Details on configuring Fluent Bit to your logging provider can be found in the documentation [here](https://docs.vmware.com/en/VMware-Tanzu-Kubernetes-Grid/1.3/vmware-tanzu-kubernetes-grid-13/GUID-extensions-logging-fluentbit.html).
+- [**VM-based deployment**](https://goharbor.io/docs/2.1.0/install-config/installation-prereqs/) - using docker-compose - this installation method is recommended in cases where Tanzu Kubernetes Grid is installed in an air-gapped environment and no pre-existing Kubernetes clusters exist on which to install Harbor.
 
-The Fluent Bit `DaemonSet` is installed automatically as an extension on all Tanzu Kubernetes Grid clusters, but each Tanzu Kubernetes Grid cluster can be configured differently if desired.
+![Harbor Container Registry](img/tko-on-vmc-aws/tko-vmc-aws06.png)
 
-vRealize Log Insight (vRLI) provides real-time log management and log analysis with machine learning based intelligent grouping, high-performance searching, and troubleshooting across physical, virtual, and cloud environments. vRLI already has a deep integration with the vSphere platform, where you can get key actionable insights, and it can be extended to include the cloud native stack as well.
+#
+# vSphere with Tanzu SaaS Integration
 
-The vRealize Log Insight appliance is available as a separate on-premises deployable product. You can also choose to go with the SaaS version vRealize Log Insight Cloud.
+The SaaS products in the VMware Tanzu portfolio are in the critical path for securing systems at the heart of your IT infrastructure. VMware Tanzu Mission Control provides a centralized control plane for Kubernetes, and Tanzu Service Mesh provides a global control plane for service mesh networks. Tanzu Observability provides features like Kubernetes monitoring, Application observability and Service insights.
 
+To learn more about TKG integration with Tanzu SaaS, please see [Tanzu SaaS Services](tko-saas.md)
 
-## Summary
+# Summary
 
 Tanzu on VMware Cloud on AWS offers high-performance potential, convenience, and addresses the challenges of creating, testing, and updating Kubernetes platforms in a consolidated production environment. This validated approach will result in a production quality installation with all the application services needed to serve combined or uniquely separated workload types via a combined infrastructure solution.
 
 This plan meets many Day 0 needs for aligning product capabilities, such as configuring firewall rules, networking, load balancing, and workload compute, to the full stack infrastructure.
 
 <!-- Deployment Instructions -->
+
 <!-- For information on how to deploy this reference design, see . -->
 
 ## <a id=appendix-a></a> Appendix A
```

Comment:
```suggestion

```
Removed this heading and link target because all the link references above were removed.  The remainder of this document should be deleted, but I can't do it here because GitHub won't allow it.